hack/lib/test.sh

```diff
@@ -246,3 +246,78 @@ kube::test::if_supports_resource() {
   done
   return 1
 }
+
+
+kube::test::version::object_to_file() {
+  name=$1
+  flags=$2
+  file_name=$3
+  kubectl version $flags | grep "$name Version:" | sed -e s/"$name Version: version.Info{"/'/' -e s/'}'/'/' -e s/', '/','/g -e s/':'/'=/g' -e s/'"'/""/g | tr , '\n' > ${file_name}
```

Comment:
It isnt a test. Its trying to get either the client or server version of output, which should be stable since we have tests that depend on this output being correct.